lte/gateway/python/integ_tests/s1aptests/test_s1_handover.py

```diff
@@ -63,7 +63,7 @@ def test_s1_handover(self):
         # Columns: Cell Id, Tac, EnbType, PLMN Id, PLMN length
         enb_list = [
             [1, 1, 1, "00101", 5],
-            [2, 2, 1, "00101", 5],
+            [2, 1, 1, "00101", 5],
```

Comment:
what is the context for the Tac value change?

Reply:
For some reason, running the test on my machine fails at the S1 setup stage when using different TACs.
There is no reason to use different TACs for these tests I guess

Reply:
you need to run `s1aptests/test_modify_mme_config_for_sanity.py` for this to pass, this is executed before the sanity test list to update the MME config, I'd suggest to keep this unchanged, running the above s1ap test before this one should make it work